README.md

```diff
@@ -51,55 +51,3 @@ def my_workflow(x: int, y: int) -> int:
 Refer to [plugins/README.md](plugins/README.md) for a list of available
 plugins. There may be plugins outside of this list, but this list is maintained
 by the core maintainers.
```

Comment:
where did all the contrib docs go? Did we move it to contributing?

Reply:
Not yet, i copied it into a text file locally.  I was gonna work in it next.